client/src/components/App.jsx

```diff
@@ -38,7 +38,25 @@ const App = () => {
 
   return (
     <div>
-      <Header />
+      <Flex w='100vw'>
+        <Link to="/">
+          <Heading as='h1' fontSize='6vh' h='11.5vh' color='brand.400' p={8}>bored tour</Heading>
+        </Link>
+        <Spacer />
+        <Box p={4}>
+        <Menu>
+          <MenuButton padding='1vw' fontSize='2.5vh' color='brand.400' as={Button} rightIcon={<ChevronDownIcon />}>
+          Username
+          </MenuButton>
+          <MenuList fontSize='2.5vh' color='brand.400'>
+            <Link to="/events"><MenuItem>My Events</MenuItem></Link>
+            <Link to="/friends"><MenuItem>My Friends</MenuItem></Link>
+            <Link to="/interests"><MenuItem>My Interests</MenuItem></Link>
+            <Button fontSize='2.5vh' color='brand.400' onClick={logout}> Log Out </Button>
```

Comment:
These menu items (e.g., log out) will show even if the user is not logged in. 

The `header.js` file conditionally renders the right menu items based on whether the user is logged in. Could I make a PR to non-fade the homepage banner but still use `header.js` so we have the right menu items?

Reply:
reverted header in App.jsx in last commit